examples/conf-quickstart/druid/_common/common.runtime.properties

```diff
@@ -21,10 +21,10 @@
 # We are using local fs for deep storage - not recommended for production - use S3, HDFS, or NFS instead
 # We are using local derby for the metadata store - not recommended for production - use MySQL or Postgres instead
 
-# If you specify `druid.extensions.loadList=[]`, Druid won't load any extension from file system.
-# If you don't specify `druid.extensions.loadList`, Druid will load all the extensions under root extension directory.
-# More info: http://druid.io/docs/latest/operations/including-extensions.html
-druid.extensions.loadList=[]
+# This is not the full list of Druid extensions, but common ones that people often use. You may need to change this list
+# based on your particular setup.
+#druid.extensions.loadList=["kafka-eight", "oss-extensions", "histogram", "datasketches", "lookups-cached-global", "mysql-metadata-storage"]
+druid.extensions.loadList=["druid-kafka-eight", "druid-oss-extensions", "druid-histogram", "druid-datasketches", "druid-lookups-cached-global", "mysql-metadata-storage"]
```

Comment:
I think it's not the best choice to put druid-oss-extensions in the conf-quickstart, we have too many customized extensions and we can't put all of them here